ops/scripts/setup_devdb.sh

```diff
@@ -18,14 +21,21 @@ else
 	DOCKER_COMPOSE="docker-compose"
 fi
 
-# create test database if not existing
+# create database if not existing
 $DOCKER_COMPOSE run --rm test bash -c "psql -h database -U gigadb -c 'create database gigadb'" || true
 
-# generate migrations
-$DOCKER_COMPOSE run --rm csv-to-migrations bash -c "npm install /var/www/ops/scripts && node /var/www/ops/scripts/csv_yii_migration.js dev"
+# generate migrations (by default "test" data is loaded in the dev db)
+# docker run -v `pwd`:/var/www node:14.9.0-buster bash -c "node /var/www/ops/scripts/csv_yii_migration.js test"
+$DOCKER_COMPOSE run --rm js bash -c "node /var/www/ops/scripts/csv_yii_migration.js $dbSet"
 
 # and run them
 $DOCKER_COMPOSE run --rm  application ./protected/yiic migrate to 300000_000000 --connectionID=db --migrationPath=application.migrations.admin --interactive=0
 $DOCKER_COMPOSE run --rm  application ./protected/yiic migrate mark 000000_000000 --connectionID=db --interactive=0
 $DOCKER_COMPOSE run --rm  application ./protected/yiic migrate --connectionID=db --migrationPath=application.migrations.schema --interactive=0
-$DOCKER_COMPOSE run --rm  application ./protected/yiic migrate --connectionID=db --migrationPath=application.migrations.data.dev --interactive=0
+$DOCKER_COMPOSE run --rm  application ./protected/yiic migrate --connectionID=db --migrationPath=application.migrations.data.$dbSet --interactive=0
+
+# run migration for FUW database
+$DOCKER_COMPOSE exec -T console /app/yii migrate/fresh --interactive=0
+
+# Fix sequences
+$DOCKER_COMPOSE run --rm application ./protected/yiic sequencefixer fixcurationlog
```

Comment:
Add `$DOCKER_COMPOSE run --rm application ./protected/yiic sequencefixer fixfile` to fix the file id sequence during setup? This could resolve the `ERROR:  duplicate key value violates unique constraint "file_pkey"`.

Reply:
See commit rija/gigadb-website@8bacdbf1 for a more generic approach